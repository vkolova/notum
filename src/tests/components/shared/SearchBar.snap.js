/* eslint-disable */
import React from 'react'
import renderer from 'react-test-renderer';

import SearchBox from '../../../components/shared/SearchBar'

describe('<SearchBox/>', () => {
    it('renders correctly', () => {
        const tree = renderer.create(
            <SearchBox/>
        ).toJSON();
        expect(tree).toMatchSnapshot();
    });
})
