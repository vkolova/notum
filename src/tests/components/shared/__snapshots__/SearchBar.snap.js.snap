// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SearchBox/> renders correctly 1`] = `
<div
  className="search"
>
  <i
    aria-hidden="true"
    className="fa fa-search"
  />
  <input
    onChange={[Function]}
    onKeyPress={[Function]}
    placeholder="Search..."
    type="text"
  />
</div>
`;
